middleware/rewrite/README.md

```diff
@@ -23,6 +23,10 @@ rewritten; e.g., to rewrite CH queries to IN use `rewrite class CH IN`.
 When the FIELD is `name` the query name in the message is rewritten; this
 needs to be a full match of the name, e.g., `rewrite name miek.nl example.org`.
 
+When the FIELD is `edns0` an EDNS0_LOCAL option can be appended to the request;
```

Comment:
What happens when I want to rewrite the EDNS0 bufsize and friends? Should would subdivide `edns0` in to `edns0 local` or something? Scrubbing stuff out of EDNS0 might also make sense (or at least the language should not preclude it)